test/language/identifier-resolution/assign-to-globalThis-undefined.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-initializeboundname
+description: >
+  In strict mode code, re-resolve unresolvable references on the global in PutValue.
+info: |
+  via sec-putvalue
+
+  If IsUnresolvableReference(V) is true, then
+    If IsStrictReference(V) is true, then
+      Let realm be the current Realm Record.
+      Set V to ? ResolveBinding(realm.[[GlobalEnv]], name).
+      Assert: IsStrictReference(V) is true.
+      If IsUnresolvableReference(V) is true, throw a ReferenceError exception.
+
+flags: [onlyStrict]
+---*/
+
+undeclared = (globalThis.undeclared = 5);
```

Comment:
I don't understand how the `globalThis` binding is related to the normative change. By my reading, this test only differs from the existing test in that it also includes semantics that are verified by [`test/built-ins/global/global-object.js`](https://github.com/tc39/test262/blob/26f1f4567ee7e33163d961c867d689173cbb9065/test/built-ins/global/global-object.js).